docs/MO_DG/prepare_model/customize_model_optimizer/Customize_Model_Optimizer.md

```diff
@@ -10,6 +10,8 @@
 
 @endsphinxdirective
 
+> **NOTE**: This document contains description of Model Optimizer internals which should be used as a last resort. As internal API it may have changes which are not backward compatible in the future. If you are going to add support for operations from ONNX or PaddlePaddle, or you are not familiar with other extension alternatives in OpenVINO, please consider reading [this guide](../../Extensibility_UG/Intro.md) first.
```

Comment:
```suggestion
> **NOTE**: This document contains description of Model Optimizer internals which should be used as a last resort. As internal API it may have changes which are not backward compatible in the future. If you are going to add support for operations from ONNX or PaddlePaddle please read [this guide](../../Extensibility_UG/Intro.md) instead. For other frameworks continue reading this document.
```

Reply:
Yes, I like this better!

Reply:
```suggestion

This article describes Model Optimizer internals. Altering them may result in application instability and in case of future changes to the API, lack of backward compatibility.

> **NOTE**: 
If you want to add support for ONNX or PaddlePaddle operations, or you are not familiar with other extension alternatives in OpenVINO, read [this guide](../../Extensibility_UG/Intro.md) instead.
```
Just an observation - we could split the two parts of the disclaimer so that one describes the aim of the article and one stays as a disclaimer pointing out just one straightforward message about ONNX and PdPd.